src/containers.rs

```diff
@@ -26,23 +26,56 @@ use edgehog_containers::{
     store::StateStore,
     Docker,
 };
+use serde::Deserialize;
+use stable_eyre::eyre::eyre;
 use tracing::error;
 
 use crate::controller::actor::Actor;
 
+/// Configuration for the container service.
+#[derive(Debug, Clone, Copy, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct ContainersConfig {
+    /// Flag to make the container service is required
+    #[serde(default)]
+    required: bool,
+    #[serde(default = "ContainersConfig::default_max_retires")]
+    max_retires: usize,
+}
+
+impl ContainersConfig {
+    const fn default_max_retires() -> usize {
+        10
```

Comment:
Defines `10` as a constant.
```suggestion
        const MAX_RETRIES: i32 = 10;
```